frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -873,6 +878,11 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
             if (scheduledChange && scheduledChange) {
                 lemonToast.success('Change scheduled successfully')
                 actions.loadScheduledChanges()
+                actions.setFeatureFlag({
+                    ...values.featureFlag,
```

Comment:
this happens only on the keyed logic with id 'schedule' right?

Reply:
correct!